src/drivers/imx/ipc.c

```diff
@@ -200,3 +200,93 @@ int platform_ipc_init(struct ipc *ipc)
 	return 0;
 }
 
```

Comment:
@lgirdwood this looks fine. Also the use cases looks safe: gdb or bootloader. One should be careful with this API in order not to interfere with the normal IRQ path.

Reply:
GDB use case is that the exception handler should invoke a ```while(1) {process_ipc();}``` on a new C stack (can probably reserve a small ELF section for this so that original stack, etc would remain intact).
Boot loader use case is very simply, it juts waits forever for ``` FW_LOAD ``` IPC from host before invoking FW load flow.

Reply:
@lgirdwood all right thanks!